otter/log/cloudfeeds.py

```diff
@@ -188,13 +189,24 @@ def __call__(self, event_dict):
         try:
             eff = self.add_event(event_dict, self.tenant_id, self.region, log)
         except UnsuitableMessage as me:
-            log.err(None, ('Tried to add unsuitable message in cloud feeds: '
-                           '{unsuitable_message}'),
-                    otter_msg_type='cf-unsuitable-message',
+            log.err(None, 'cf-unsuitable-message',
                     unsuitable_message=me.unsuitable_message)
         else:
             return perform(
                 self.get_disp(self.reactor, self.authenticator, log,
                               self.service_configs),
-                eff).addErrback(log.err, "Failed to add event",
-                                otter_msg_type='cf-add-failure')
+                eff).addErrback(log.err, 'cf-add-failure')
+
+
+def add_cf_observer(reactor, authenticator, tenant_id, region,
+                    service_configs):
+    """
+    Add cloud feeds observer after setting up some intial formatting
+    """
+    cf_observer = CloudFeedsObserver(
+        reactor=reactor, authenticator=authenticator, tenant_id=tenant_id,
+        region=region, service_configs=service_configs)
+    addObserver(
```

Comment:
I would *slightly* prefer not calling addObserver here, but rather returning an observer. Then the deployment code could call addObserver. No big deal though.